internal/browser/seperator.go

```diff
@@ -0,0 +1,58 @@
+package browser
+
+import (
+	"fmt"
+	"strings"
+	"sync/atomic"
+
+	"github.com/charmbracelet/bubbles/spinner"
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// progressState is the global flag to show/hide progress
+var progressState int32
+
+const (
+	HideProgress = iota
+	ShowProgress
+)
+
+var spinnerWidget = spinner.New()
+
+func changeProgress(ps int32) {
+	switch {
+	case ps == HideProgress:
```

Comment:
Why not:
```
switch ps {
    case HideProgress:
// ...
```

Reply:
that's better, thanks! https://github.com/hazelcast/hazelcast-commandline-client/pull/104/commits/925164c7c36af50ed95e949a2686edda2bce66f2